scripts/install.sh

```diff
@@ -101,12 +101,13 @@ sudo apt-get update
 sudo apt-get install -y openjdk-8-jre-headless
 sudo update-java-alternatives --jre-headless -s java-1.8.0-openjdk-amd64
 
-sudo scylla_setup --no-raid-setup --no-ec2-check --no-kernel-check --no-verify-package --no-sysconfig-setup --io-setup=1 --no-version-check --no-cpuscaling-setup --no-fstrim-setup --no-memory-setup --no-swap-setup
+sudo scylla_setup --no-raid-setup --no-ec2-check --no-kernel-check --no-verify-package --no-sysconfig-setup --io-setup=1 --no-version-check --no-cpuscaling-setup --no-fstrim-setup --no-memory-setup --no-swap-setup 2>/dev/null
```

Comment:
Can you extract this into another PR ?

Reply:
I will remove this change since it doesn’t make any difference during Scylladb installation.

Reply:
ok

Reply:
I have used `sleep 5` since the `nodetool status` shows error as scylla-server server takes time to start.